src/test/java/org/assertj/core/api/string_/StringAssert_isEqualTo_Test.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * Copyright 2012-2018 the original author or authors.
+ */
+package org.assertj.core.api.string_;
+
+import static org.mockito.Mockito.verify;
+
+import org.assertj.core.api.StringAssert;
+import org.assertj.core.api.StringAssertBaseTest;
+
+public class StringAssert_isEqualTo_Test extends StringAssertBaseTest {
+
+  @Override
+  protected StringAssert invoke_api_method() {
+    return assertions.isEqualTo("%s and %d", 'A', 8);
+  }
+
+  @Override
+  protected void verify_internal_effects() {
+    verify(comparables).assertEqual(getInfo(assertions), getActual(assertions), "A and 8");
```

Comment:
it looks like you have to adjust this one to your recent changes -> see pr build on travis.

Reply:
@epeee not clear

Reply:
oh sorry, yes. I see now. It's done.